### PR TITLE
[docs] 1st PR for Restructure Explore Part 2 (Doc9434)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Checking in list items 1, 2, and 6 of https://github.com/yugabyte/yugabyte-db/issues/9434
Reassigning the base issue to Alex to finish the rest.

### CODE REVIEW

File: docs/content/latest/explore/cluster-management/_index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ menu:
   latest:
```
Comment (polarweasel) [inline docs/content/latest/explore/cluster-management/_index.md:13]: @lizayugabyte pls delete the aliases section in this file. It conflicts with `point-in-time-recovery-ycql.md`

**Review by polarweasel — CHANGES_REQUESTED** [review]
Just one change (it slipped in in the PR that added cluster-management)...

File: docs/content/latest/explore/cluster-management/_index.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/explore/cluster-management/_index.md:9]: Deleted

**Review by polarweasel — APPROVED** [review]
LGTM!

**Pull request merged**